localizer_psychopy/localizer_exp

```diff
@@ -0,0 +1,81 @@
+# import psychopy modules
+from psychopy import visual, core, event, sound, gui, data
+
+#set parent directory
+parent_dir = "/Users/Kayleigh/fmriproject/localizer_psychopy/"
+
+# Set up text displays
+
+win = visual.Window(size = [1920,1080],
+                    color = "white",
+                    fullscr = False,
+                    units = "pix")
+
+Rhand_txt = visual.TextStim(win, text = "Curl your right hand.",
+                        pos = [0,0],
+                        color = "black",
+                        height = 50,
+                        font = "Arial")
+
+Lhand_txt = visual.TextStim(win, text = "Curl your left hand.",
+                        pos = [0,0],
+                        color = "black",
+                        height = 50,
+                        font = "Arial")
+
+Rfoot_txt = visual.TextStim(win, text = "Curl your right foot.",
+                        pos = [0,0],
+                        color = "black",
+                        height = 50,
+                        font = "Arial")
+
+Lfoot_txt = visual.TextStim(win, text = "Curl your left foot.",
+                        pos = [0,0],
+                        color = "black",
+                        height = 50,
+                        font = "Arial")
+
+tongue_txt = visual.TextStim(win, text = "Curl your tongue.",
+                        pos = [0,0],
+                        color = "black",
+                        height = 50,
+                        font = "Arial")
+
+null_txt = visual.TextStim(win, text = "+",
+                        pos = [0,0],
+                        color = "black",
+                        height = 50,
+                        font = "Arial")
+
+
+# in the data source, there are two columns: Time (0-502) and StimType (1-6)
+# this part is trying to link the numbers in StimType to the names of the text displays
+STIM = {1: Rhand_txt, 2: Lhand_txt, 3: Rfoot_txt, 4: Lfoot_txt, 5: tongue_txt, 6: null_txt}
+# load in our stimulus timing csv file
+TRIAL_LIST = data.importConditions(fileName = parent_dir + "localizer_datasource.csv")
+
+#create clock
+globalClock = core.Clock()
+
+# start clock from 5 from scanner --- tim is going to add this part
+
+# rename clock?
+t0 = globalClock.getTime()
+
+# while the clock is running...
+while t0 >= 0:
```

Comment:
i would do `while TRUE` instead, but it might not really matter in this case
you can also do `while global.getTime()-t0 < 503.25` (or whatever upper limit of time you wanna set for this experiment)